src/test/java/uk/gov/hmcts/reform/civil/handler/tasks/GenerateCsvAndTransferHandlerTest.java

```diff
@@ -129,6 +136,21 @@ void should_handle_task_from_external_variable() {
         verify(externalTaskService).complete(externalTask);
     }
 
+    @Test
+    void shouldGenerateCsvAndSendEmailSuccessfully_R2LipVLipFlagEnabled() {
+        when(toggleService.isLipVLipEnabled()).thenReturn(true);
+        when(mediationCsvServiceFactory.getMediationCSVService(any())).thenReturn(mediationCSVLipvLipService);
+        when(searchService.getInMediationCases(claimToBeProcessed)).thenReturn(List.of(caseDetailsWithInMediationState));
+        when(caseDetailsConverter.toCaseData(caseDetailsWithInMediationState)).thenReturn(caseDataInMediation);
+        when(caseDetailsConverter.toCaseData(caseDetailsWithInMediationStateNotToProcess)).thenReturn(caseDataInMediationNotToProcess);
+
+        inMediationCsvHandler.execute(externalTask, externalTaskService);
+        verify(searchService).getInMediationCases(claimToBeProcessed);
```

Comment:
Add times in all verifications

Reply:
All test cases already having this check